recipes/miniz/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import glob
+import os
+import shutil
+
+from conans import ConanFile, CMake, tools
+
+class MinizConan(ConanFile):
+    name = "miniz"
+    description = "Lossless, high performance data compression library that " \
+                  "implements the zlib (RFC 1950) and Deflate (RFC 1951) " \
+                  "compressed data format specification standards"
+    license = "MIT"
+    topics = ("conan", "miniz", "compression", "lossless")
+    homepage = "https://github.com/richgel999/miniz"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
```

Comment:
Consider adding `header_only` and `amalgamate` as options. https://github.com/richgel999/miniz/blob/master/CMakeLists.txt#L26

Reply:
It is not available in 2.1.0.

Reply:
Now I see, it's a complete different project due the version.